Projects/Claims/Sources/Models/FlowClaimSingleItemCheckoutStepModel.swift

```diff
@@ -50,4 +50,20 @@ struct ClaimFlowMoneyModel: Codable, Equatable {
         formatter.currencySymbol = self.currencyCode
         return formatter.string(for: amount) ?? ""
     }
+
+    func formatAndPrintDouble(value: Double) -> String {
```

Comment:
maybe we should rename this function to something like
func getValueWIthCurrency() -> String